params-local-folder.json

```diff
@@ -0,0 +1,18 @@
+{
+  "seq_reads_folder": "test/samples/",
+  "sample_in_metadata": "test/samples/in-metadata.tsv",
+  "vector_type": "unspecified",
+  "vector_bed": "test/samples/sc.annotation.bed",
+  "vector_fa": "test/samples/sc.construct.fasta",
+  "packaging_fa": "test/fasta/packaging.fa",
+  "host_fa": "test/fasta/hg38.chr19trunc-chrM.fa",
+  "itr_label_1": "ITR-L",
+  "itr_label_2": "ITR-R",
+  "repcap_name": "pRep2Cap9",
+  "helper_name": "pHelper",
+  "lambda_name": "Lambda",
+  "target_gap_threshold": 200,
+  "max_allowed_outside_vector": 10,
+  "max_allowed_missing_flanking": 70,
```

Comment:
Do you compare results for expected results with max_allowed_missing_flanking of 70?